backend/open_webui/apps/audio/main.py

```diff
@@ -32,7 +34,7 @@
 )
 
 from open_webui.constants import ERROR_MESSAGES
-from open_webui.env import ENV, SRC_LOG_LEVELS, DEVICE_TYPE
```

Comment:
Unwanted introduced with from resolving conflicts.

Reply:
> Unwanted changes introduced with from resolving conflicts.

Sorry, but I don't understand what the problem is. Everything works for me without errors. I'm not a professional programmer. I just gave you an idea. And the implementation that works for me. I am attaching a file that works for me open-webui-0.3.35/backend/open_webui/apps/audio/
[main.py.txt](https://github.com/user-attachments/files/17736065/main.py.txt)